pkg/webbootDhclient/webboot_dhclient.go

```diff
@@ -0,0 +1,74 @@
+package webbootDhclient
+
+import (
+	"context"
+	"log"
+	"regexp"
+	"time"
+
+	"github.com/u-root/u-root/pkg/dhclient"
+	"github.com/vishvananda/netlink"
+)
+
+func DhclientNew(ifName string, timeout int, retry int, verbose bool, ipv4 bool, ipv6 bool) {
```

Comment:
just call this New. Just call the file dhclient.go. Just make the directory name dhclient. 
Make all file names all lower case.
So 
New()

Reply:
Resolved :)! Good point!

Reply:
I later changed the function name from New() to Request because one can expect this function to create a new instance of dhclient, not to do the whole protocol if it is called New()